.test-infra/jenkins/job_Release_Python_NightlySnapshot.groovy

```diff
@@ -41,11 +41,19 @@ job('beam_Release_Python_NightlySnapshot') {
             'Run Python Publish')
 
     steps {
+      // Cleanup Python directory.
       gradle {
         rootBuildScriptDir(commonJobProperties.checkoutDir)
-        tasks(':beam-sdks-python:sdist')
+        tasks(':beam-sdks-python:clean')
```

Comment:
Could we combine these tasks (clean, buildSnapshot) into one single gradle wrapper?

Reply:
I prefer to keep `buildSnapshot` and `clean` independent at Gradle task level. If you mean to combine two Gradle call to one in this file, I think we could add `mustRunAfter clean` to `buildSnapshot` Gradle config and do something here like:
```
      gradle {
        rootBuildScriptDir(commonJobProperties.checkoutDir)
        tasks(':beam-sdks-python:clean')
        tasks(':beam-sdks-python:buildSnapshot')
        commonJobProperties.setGradleSwitches(delegate)
      }
```

Reply:
looks like `clean` and `buildSnapshot` shouldn't run from single context since virtualenv directory will be cleanup right after `clean` and `buildSnapshot` depends on it.